cmd/maya-agent/app/block_devices_test.go

```diff
@@ -0,0 +1,23 @@
+package app_test
+
+import (
+	"fmt"
+	"os/exec"
+	"testing"
+)
+
+func TestListBlockDevice(t *testing.T) {
+	res, err := exec.Command("../maya-agent", "block", "list").Output()
+	if err != nil {
```

Comment:
You are not validating the expected output, we should define a disk schema before all of this